packages/console/src/monaco/MonacoUtils.ts

```diff
@@ -363,7 +415,9 @@ class MonacoUtils {
       keybindings.forEach(keybinding =>
         MonacoUtils.removeKeybinding(
```

Comment:
Check if the keybinding is empty before calling `removeKeyBinding` ... then you don't need the check for empty str in `MonacoUtils.removeKeybinding`.